analysis/preprocess/Stage1_data_cleaning.R

```diff
@@ -321,6 +322,21 @@ stage1 <- function(cohort_name){
     #----------------------#
     write.csv(cohort_flow, file = file.path("output/review/descriptives", paste0("Cohort_flow_",cohort_name, ".csv")) , row.names=F)
     
+    #--------------------------#
+    # 3.e. Generate histograms #
+    #--------------------------#
+    # generate histograms for numerical variables
+    
+    numeric_vars <- input %>% dplyr::select(contains("_num"))
+    numeric_title <- colnames(numeric_vars)
+    
+    svglite::svglite(file = file.path("output/not-for-review/", paste0("numeric_histograms_", cohort_name, ".svg")), width = 15, height = 8)
+    g <- ggplot(gather(numeric_vars), aes(value)) + 
+      geom_histogram(bins = 10) + 
+      facet_wrap(~key, scales = 'free_x')
+    print(g)
+    dev.off()
+    
```

Comment:
The histogram looks good locally. But I lost track with the variable "cov_num_bmi_date_measured" - looks like a date variable?

Reply:
This has now been renamed in the preprocess script to "tmp_out_bmi_date_measured" as has been done in the unvaccinated repo.